signalfx-java/src/main/java/com/signalfx/metrics/connection/AbstractHttpDataPointProtobufReceiverConnection.java

```diff
@@ -50,7 +50,8 @@ public void addDataPoints(String auth, List<SignalFxProtocolBuffers.DataPoint> d
                         compress);
 
                 int code = resp.getStatusLine().getStatusCode();
-                if (code != HttpStatus.SC_OK) {
+                // Signalfx backend considers any 2xx return code success.
```

Comment:
"Considers" doesn't seem right for a response. How about the following:

```java
// SignalFx may respond with various 2xx return codes for success.
```

Reply:
Will change very soon

Reply:
Done.